rmw_zenoh_cpp/src/detail/event.cpp

```diff
@@ -12,6 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <deque>
+#include <memory>
```

Comment:
Do we need to specify these headers again if they're included in the header file?

Reply:
> Do we need to specify these headers again if they're included in the header file?

My general thinking here is that yes, if the implementation uses the signatures, it should also include it.  Essentially I believe in "include-what-you-use" in both the header file and the implementation file.

Reply:
Gotcha